tests/core/test_cells.py

```diff
@@ -90,12 +80,12 @@ def test_cell_get_cell():
     assert isinstance(hexbeam.get_cell(0), pyvista.Cell)
 
 
-@pytest.mark.parametrize("cell", cells[:5])
+@pytest.mark.parametrize("cell", cells)
```

Comment:
```suggestion
@pytest.mark.parametrize("cell", cells, ids=types)
```

Reply:
I chose to use the original `cell_ids` instead.  Leaving this open in case there are other opinions.